tests/integration/src/utils/fluent_bit_manager.py

```
@@ -129,58 +255,163 @@ def stop(self):
+    def send_sighup(self):
+        self.send_signal(signal.SIGHUP)
+
+    def get_reload_status(self):
+        url = f"http://127.0.0.1:{self.http_monitoring_port}/api/v2/reload"
+        response = requests.get(url, timeout=0.5)
+        response.raise_for_status()
+        return response.json()
+
+    def trigger_http_reload(self):
+        url = f"http://127.0.0.1:{self.http_monitoring_port}/api/v2/reload"
+        response = requests.post(url)
```

Ruff (0.15.21)

[error] Probable use of `requests` call without timeout

(S113)